Sources/Plasma/NucleusLib/pnAsyncCore/Private/pnAcThread.h

```diff
@@ -65,10 +66,39 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 // for IoWaitId/TimerCreate/TimerUpdate
 constexpr unsigned kAsyncTimeInfinite = (unsigned) -1;
 
+struct AsyncThread;
+
+struct AsyncThreadRef {
+    std::shared_ptr<AsyncThread> impl;
+    std::thread&                 thread() const;
+    bool joinable() const;
+};
+
+
+// Threads are also allowed to set the workTimeMs field of their
+// structure to a nonzero value for "on", and IO_TIME_INFINITE for
+// "off" to avoid the overhead of calling these functions. Note
+// that this function may not be called for the main thread. I
+// suggest that application code not worry that timeMs might
+// "accidentally" equal the IO_TIME_INFINITE value, as it only
+// happens for one millisecond every 49 days.
+struct AsyncThread {
+    LINK(AsyncThread)                    link;
```

Comment:
Are threads still placed into an intrusilvely linked list as implemented by `pnUtList`?

Reply:
No, I'll eliminate that struct member.